README.md

```diff
@@ -8,7 +8,7 @@
 
 [![Build and Test][7]][8] [![JitPack build status][9]][10] [![Code coverage][11]][12]
 
-[![TODOs][13]][14] [![LGTM][15]][16] [![REUSE status][17]][18] [![CII][19]][20]
+[![TODOs][13]][14] [![REUSE status][15]][16] [![CII][17]][18]
```

Comment:
Please remove the changes to this file from this commit.

Reply:
...and do so again by amending this commit, not by creating another commit in this PR, please.

Reply:
Done